cartesian_impedance_msgs/msg/CartesianDamping.msg

```diff
@@ -0,0 +1,11 @@
+# Damping parameters for the Cartesian Impedance
+# the values are divided into translational and rotational
+# each of the translational and rotational components is defined as Vector3
+# for more info on Vector3 messages please see the following link:
+# http://docs.ros.org/api/geometry_msgs/html/msg/Vector3.html
+
+
+# [N/m]
+geometry_msgs/Vector3 translational
+# [Nm/rad]
+geometry_msgs/Vector3 rotational
```

Comment:
Yes you are right Shaun. Is dampening about the fixed frame axes. These values are usually consistent and each value during the motion is constant, so there are no singularities that come with Euler angles actually these are just constants which are used to define the properties of the Spring when the robot is in Cartesian Impedance Mode.